docs/en/latest/references/apisix_cluster_config_v2.md

```diff
@@ -42,3 +42,4 @@ See the [definition](https://github.com/apache/apisix-ingress-controller/blob/ma
 | admin                             | object  | Admin configurations.                          |
 | admin.baseURL                     | string  | Base URL of the APISIX cluster.                |
 | admin.AdminKey                    | string  | Admin key to authenticate with APISIX cluster. |
+| prefer_name                       | boolean | When set to `true`, prints Route/Service name instead of their ID in Prometheus metrics. |
```

Comment:
This is a property of `monitoring.prometheus`. See: https://github.com/apache/apisix-ingress-controller/blob/6028b62bb3d644891fdb3079c6de3e4303f2430c/samples/deploy/crd/v1/ApisixClusterConfig.yaml#L65

Reply:
good catch!